1-exercises/F-for-of-loop/exercise.js

```diff
@@ -11,6 +11,17 @@ let tubeStations = [
     "Tottenham Court Road"
 ];
 
```

Comment:
With a for .. of loop, it would be 
for (<type> elements of <object>) {
 }. 
<type> could be int, const, string or BigInt